src/lib.rs

```diff
@@ -6,6 +6,8 @@ extern crate libc;
 #[macro_use]
 pub mod syslog;
 
+/// Macros and functions for working with
+/// [`ioctl`] (http://man7.org/linux/man-pages/man2/ioctl.2.html)
```

Comment:
There is an extra space between the URL name and link.

Please run `cargo doc` to make sure that the documentation is properly formatted.